components/utils/time.js

```diff
@@ -1,4 +1,4 @@
-export default {
+const time = {
```

Comment:
I think this is just a matter of code style. Any particular reason to move the exposing to an object?